app/views/bets/_display_bets.html.erb

```diff
@@ -17,7 +17,15 @@
             <td class="col-md-2"><%= bet.created_at %></td>
             <td class="col-md-2"><%= bet.expires_at %></td>
             <td class="col-md-2"><%= bet.bet_total %></td>
-            <td class="col-md-4"><%= bet.bet %></td>
+            <td class="col-md-3"><%= bet.bet %></td>
+            <td class="col-md-1">
+              <%=
+                link_to bet, class: 'btn btn-secondary', role: 'button' do
+                  content_tag :i, class: 'fa fa-eye' do
+                  end
```

Comment:
Yeah, it bombs without the _do...end_. Didn't know you could one line it with passing _nil_ though, thanks!
Now, if you excuse me, I have 5 thousand other "content_tag"s to fix.